### ISSUE TITLE
[gitops] the server could not find the requested resource

### ISSUE THREAD

**Issue by Hongbo-Miao** [linkerd/linkerd-examples#265]
I am following the guide at [Using GitOps with Linkerd with Argo CD](https://linkerd.io/2.10/tasks/gitops/) step by step.
(I have fixed one issue at https://github.com/linkerd/linkerd-examples/pull/264)
Now I met a this new error below when running `argocd app sync linkerd-bootstrap`

```shell
TIMESTAMP                  GROUP                  KIND     NAMESPACE                  NAME       STATUS    HEALTH        HOOK  MESSAGE
2021-06-26T06:50:29+08:00  cert-manager.io      Issuer       linkerd  linkerd-trust-anchor     OutOfSync  Missing              the server could not find the requested resource
2021-06-26T06:50:29+08:00  cert-manager.io  Certificate      linkerd  linkerd-identity-issuer  OutOfSync  Missing              the server could not find the requested resource
2021-06-26T06:50:29+08:00                    Namespace                             linkerd     OutOfSync
2021-06-26T06:50:29+08:00  bitnami.com      SealedSecret     linkerd  linkerd-trust-anchor     OutOfSync  Missing

Name:               linkerd-bootstrap
Project:            demo
Server:             https://kubernetes.default.svc
Namespace:          linkerd
URL:                https://127.0.0.1:8080/applications/linkerd-bootstrap
Repo:               git://git-server.scm:9418/linkerd-examples.git
Target:
Path:               gitops/resources/linkerd
SyncWindow:         Sync Allowed
Sync Policy:        <none>
Sync Status:        OutOfSync from  (da2c6b0)
Health Status:      Missing

Operation:          Sync
Sync Revision:      da2c6b0a4c491194ca845af5c5a9ec5842a07631
Phase:              Failed
Start:              2021-06-26 06:50:29 +0800 CST
Finished:           2021-06-26 06:50:29 +0800 CST
Duration:           0s
Message:            one or more synchronization tasks are not valid

GROUP            KIND          NAMESPACE  NAME                     STATUS     HEALTH   HOOK  MESSAGE
cert-manager.io  Issuer        linkerd    linkerd-trust-anchor     OutOfSync  Missing        the server could not find the requested resource
cert-manager.io  Certificate   linkerd    linkerd-identity-issuer  OutOfSync  Missing        the server could not find the requested resource
                 Namespace                linkerd                  OutOfSync
bitnami.com      SealedSecret  linkerd    linkerd-trust-anchor     OutOfSync  Missing
FATA[0000] Operation has completed with phase: Failed
```

**Comment by JasonMorgan** [comment]
Hey @Hongbo-Miao! Sorry about the delay, I'm looking through the guide now and I'll get back to you once it's updated.

**Comment by Hongbo-Miao** [comment]
Really appreciate!

**Comment by sbathgate** [comment]
I pulled my hair out over this for the last while. For those in a similar position, it looks like ArgoCD updated their version of cert-manager. If you update `cert-manager.yaml` `spec.targetRevision: v0.15.0`-> `spec.targetRevision: v1.0.4` it appears to work. 

Reference [link](https://cd.apps.argoproj.io/applications/cert-manager?resource=&node=argoproj.io%2FApplication%2Fargocd%2Fcert-manager%2F0)

I will also submit a PR shortly with the intended fix too.

**Comment by adleong** [comment]
Fixed by #266
